game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -999,6 +1049,20 @@ public Component getListCellRendererComponent(final JList<? extends Unit> list,
     }
   }
 
+  /**
+   * Prompts the user to select a collection of dice rolls. The user is presented with a dialog that allows them to
+   * choose from all possible rolls for a die with the specified number of sides. Each roll is rendered depending on
+   * whether it will result in a hit or a miss.
+   *
+   * @param numDice The number of dice the user is required to select.
+   * @param hitAt The value at which the roll is considered a hit.
+   * @param hitOnlyIfEquals {@code true} if a roll is only considered a hit if it is exactly equal to {@code hitAt};
+   *        {@code false} if any roll equal to or greater than {@code hitAt} should be considered a hit.
```

Comment:
That's what I thought, too, but the `DiceChooser` code works as documented above:

https://github.com/triplea-game/triplea/blob/6d8b537a64e348a604b2395267e72ab2db82c143/game-core/src/main/java/games/strategy/triplea/ui/DiceChooser.java#L57-L58

And here's a screenshot of running `selectFixedDice()` (`hitAt == 3` and `hitOnlyIfEquals == false`):

![dice-chooser](https://user-images.githubusercontent.com/4826349/44918495-2319bb80-ad09-11e8-93ee-2675e0be5a72.png)

I don't know if that's a bug or as designed.

Reply:
@ssoloff This was bugging me, so I started my IDE the first time since a month or so (really should be doing other things right now D: ).
And found out why this is 💡 
Looks like the only usages out there (in `TechnologyDelegate` and `ConductBombing`), 2 for Technology rolls, 1 for bombing raids all have "exact roll" set to true, meaning this method is only being used to test for exact rolls.
I'll create a small PR real quick.